targets/python/source/source/PlayFabHTTP.py

```diff
@@ -4,13 +4,25 @@
 import requests
 
 class PlayFabJsonError():
-    def __init__(self, errorMessage):
-        self.code = 0
-        self.status = ""
-        self.error = ""
-        self.errorCode = 0
-        self.errorMessage = errorMessage
-        self.errorDetails = None
+    def __init__(self, *args):
+        if args.count == 1:
+            self.fromJson(args[0])
+        else:
+            self.code = 0
+            self.status = ""
+            self.error = ""
+            self.errorCode = 0
+            self.errorMessage = errorMessage
+            self.errorDetails = None
+
+    def fromJson(self, other):
+        print(other._content.decode("utf-8"))
+        self.code = other.status_code
+        self.status = other._content["status"]
```

Comment:
Shouldn't this just be
self.status = other.status
?
(ditto for other lines)

Reply:
Ah, I should pass other._content instead and then we can do self.x = other.x for all parameters.